packages/mds-orm/entities/types/index.ts

```diff
@@ -8,6 +8,4 @@ export type JsonObject = {
   [property: string]: Nullable<JsonValue>
 }
 
-export type JsonArray = Nullable<JsonValue>[]
-
-export type JsonValue = string | number | boolean | JsonObject | JsonArray
+export type JsonValue = string | number | boolean | Record<string, any> | any[] | null
```

Comment:
Ok maybe this wasn't the best way to fix it, but there's a recursive type issue which was causing a compilation failure when I added the policy metadata test.

Reply:
I will look at it after this gets merged.